lib/puppet_x/cisco/aci_provider_utils.rb

```diff
@@ -199,112 +200,111 @@ def self.set_property_flush_attrs_frm_resource(p_inst)
+              debug_what_changed(resp_attrs_hash,
+                                 provider_inst.property_hash)
             end
           when :absent
-            resp = Puppet::Util::NetworkDevice.current.delete(
-                      self.gen_url4instance(provider_inst.property_flush[:dn]))
-          end
-          #Return the corresponding APIC GUI URI that will be used for informational
-          #message
-          self.gen_url4apicgui(provider_inst.property_flush[:fvtenant],
-            provider_inst.property_flush[:dn])
-        rescue Exception => e
-            puts e.message
-            puts e.backtrace.inspect
+            Puppet::Util::NetworkDevice.current.delete(
+              gen_url4instance(provider_inst.property_flush[:dn]))
           end
+          # Return the corresponding APIC GUI URI that will be used for informational
+          # message
+          gen_url4apicgui(provider_inst.property_flush[:fvtenant],
+                          provider_inst.property_flush[:dn])
+        rescue => e
+          puts e.message
+          puts e.backtrace.inspect
+        end
       end
 
       #=============ACI Factor related utility methods =======================
 
-      #Extract Fabric Members from  DHCP client query request.
-      #@param [JSON] Response from the DHCP Client query
-      #@return [Hash] {NodeID => Attribute Map} for leaf and spine members
+      # Extract Fabric Members from  DHCP client query request.
+      # @param [JSON] Response from the DHCP Client query
+      # @return [Hash] {NodeID => Attribute Map} for leaf and spine members
       #               Please note that there is a possibility that this method
       #               can return a nil
       #
       def self.xtract_fabric_member_frm_dhcp_query(one_dhcp_clnt_json)
-        if (one_dhcp_clnt_json and
-          (values = one_dhcp_clnt_json['dhcpClient']) and
-          (attrs  = values['attributes']) and
-          (id = attrs['id']) and
-          ((attrs['nodeRole'] == 'leaf') or (attrs['nodeRole'] == 'spine')))
-          #create an NodeID => Node Details Hash
-          {id=>attrs}
+        if one_dhcp_clnt_json &&
+           one_dhcp_clnt_json['dhcpClient'] &&
+           one_dhcp_clnt_json['dhcpClient']['attributes'] &&
+           one_dhcp_clnt_json['dhcpClient']['attributes']['id'] &&
+           (one_dhcp_clnt_json['dhcpClient']['attributes']['nodeRole'] == 'leaf' ||
+            one_dhcp_clnt_json['dhcpClient']['attributes']['nodeRole'] == 'spine')
+          # create an NodeID => Node Details Hash
+          attrs = one_dhcp_clnt_json['dhcpClient']['attributes']
+          { id => attrs }
         end
       end
 
-
-      #Generate Facts for ACI Fabric from the APIC
+      # Generate Facts for ACI Fabric from the APIC
       #
-      #@return [Hash] Facts for the ACI Fabric
+      # @return [Hash] Facts for the ACI Fabric
       #
-      def self.get_facts()
+      def self.facts
```

Comment:
Why the method name change?

Reply:
Rubocop didn't like methods starting with get_ and set_ .